### ISSUE TITLE
Cli Application doesn't work.

### ISSUE THREAD

**Issue by ciihhhddh** [TheNexusAvenger/Enigma#3]
When I load and allow the CLI file for enigma on my PC, the terminal just flashes at me with a whole bunch of red texts for .1 of a second and then closes. I was able to get this screenshot of it though.
![Screenshot 2024-07-08 234302](https://github.com/TheNexusAvenger/Enigma/assets/113849678/7d5fadcf-1f02-4a09-9e48-065adff3f3d3)

**Comment by ciihhhddh** [comment]
BTW, i've searched countless times for a tutorial but there seems to be no sign of your product anywhere else on the internet.

**Comment by noblereign** [comment]
it'd really help if you could get a clearer screenshot using a screen recorder or something, this is blurry to the point of being unreadable 😭

**Comment by ciihhhddh** [comment]
> it'd really help if you could get a clearer screenshot using a screen recorder or something, this is blurry to the point of being unreadable 😭

No, it was a screenshot. It's just that I wasnt too sure if it had any vital information that I probably wouldn't like getting leaked. I dont work on terminal really often so I wasn't sure. My apologies.

**Comment by ciihhhddh** [comment]
![Screenshot 2024-07-09 001102](https://github.com/TheNexusAvenger/Enigma/assets/113849678/6f3d9454-6fe5-4eb7-b41a-d706e6f9fc37)

**Comment by ciihhhddh** [comment]
> Are you able to scroll up to the top of the logs and post what is there, along with the last 2 or so lines? The stack trace is all internal to .NET stuff that I can't figure out.
> 
> > BTW, i've searched countless times for a tutorial but there seems to be no sign of your product anywhere else on the internet.
> 
> Enigma is _super_ new to the point no tutorial exists, yet.

It instantly does that right as I run steam vr and detects the headset.

**Comment by ciihhhddh** [comment]
That's all I know for now, I cant just scroll up because it apears for a 5th of a second.

**Comment by TheNexusAvenger** [comment]
Can you try to run the application by running it from a command prompt/terminal? Also, can you try this on V.0.4.0? There is a possibility it is not copying `openvr_api.dll`, which gave a lengthy error instead of a human-readable error. (It also probably makes whatever trackers you have actually work, because that was a problem pre-V.0.3.1.)

**Comment by ciihhhddh** [comment]
Alright, so what command should I put in the terminal?

**Comment by TheNexusAvenger** [comment]
`enigma-cli.exe` can be run with nothing extra to run. You can add `--debug` for more logging, although it may not give much more with this error.

**Comment by ciihhhddh** [comment]
> `enigma-cli.exe` can be run with nothing extra to run. You can add `--debug` for more logging, although it may not give much more with this error.

![image](https://github.com/user-attachments/assets/acfdd1ec-4925-4987-992a-6fbdd6c27806)

**Comment by TheNexusAvenger** [comment]
Your command prompt is not in the right directory to run the program.
You might want to find a guide on how to run an executable from the command prompt. They can probably explain it better than I could.

**Comment by ciihhhddh** [comment]
> Your command prompt is not in the right directory to run the program. You might want to find a guide on how to run an executable from the command prompt. They can probably explain it better than I could.

alright here it is, i succsessfuly ran it through terminal but it does the same thing. 
![image](https://github.com/user-attachments/assets/5088999e-9003-4d17-b805-431558d4ba5a)

**Comment by ciihhhddh** [comment]
I figured out the problem, my config folder that it's been trying to find is in my other SSD storage.

**Comment by TheNexusAvenger** [comment]
[I have just created V.0.4.1](https://github.com/TheNexusAvenger/Enigma/releases/tag/V.0.4.1) that should patch this error, **but does not fix the underlying cause**. Enigma currently assumes the location of this config folder, but it doesn't check any other place. I couldn't find online how you are meant to figure this out; they just give me the path that Enigma already assumes.

Because of that, V.0.4.1 will get you working, but I am renaming the issue and leaving it open until the root cause is resolved.

**Comment by ciihhhddh** [comment]
when will you be able to add hip tracking?

**Comment by ciihhhddh** [comment]
PS everything works fine! 👍

**Comment by TheNexusAvenger** [comment]
> when will you be able to add hip tracking?

Hip trackers ("Waist" in OpenVR terminology) are supported by Enigma, but not by Nexus VR Character Model. Nexus VR Character Model questions should be separate GitHub Issues for that project, but to answer that here: it is *not* planned due to no clear way to implement them in V3, and the math being far beyond what I can handle for V2.
